decode-ways/easyone-jwlee.go

```diff
@@ -0,0 +1,33 @@
+// 풀이
+// dp로 풀이
+// 두자리 수에 적합하면 prev2(i-2)에 해당하는 값을 더하기
+
+// TC
+// O(n)
+
+// SC
+// data type이 int인 변수만 사용했으므로 O(1)
```

Comment:
사소한 피드백입니다 :)
제 생각엔 조금 더 정확한 표현을 하자면 아래처럼 쓰는게 더 맞을 것 같아요
```suggestion
// n: input으로 주어진 string `s`의 길이
// 해당 알고리즘의 추가적인 공간 사용량은 n에 상관 없이 일정하므로 O(1)
```

Reply:
피드백 감사합니다. 앞으로는 좀 더 정확한 표현으로 적어보겠습니다^ㅡ^